lib/fpm/package/cpan.rb

```diff
@@ -150,6 +150,10 @@ def input(package)
        found_dependencies.each do |dep_name, version|
           # Special case for representing perl core as a version.
           if dep_name == "perl"
+            m = version.match(/^(\d)\.(\d{3})(\d{3})$/)
+            if m
+               version = m[1] + '.' + m[2].sub(/^0*/, '') + '.' + m[3].sub(/^0*/, '')
```

Comment:
will this work for all target packages? or just rpm?

If it focuses specific to rpm, then I think this can be added to the `converted_from` method in rpm.rb instead? Thoughts?